testing/ratpoison/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Duncan Guthrie <dguthrie@posteo.net>
+# Maintainer: Duncan Guthrie <dguthrie@posteo.net>
+pkgname=ratpoison
+pkgver=1.4.9
+pkgrel=0
+pkgdesc="ratpoison: Say good-bye to the rodent"
+url="http://ratpoison.nongnu.org"
+arch="all"
+license="GPL2"
+depends="libx11"
+makedepends="libx11-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="http://download.savannah.nongnu.org/releases/$pkgname/$pkgname-$pkgver.tar.xz"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!check"
```

Comment:
We normally add a check function to do a minimal verification that software runs. It may be as simple as just print the version number or help. If this is not possible, then it would be nice to have a comment on why the check was disabled.